_input/listbox-button/index.html

```diff
@@ -186,6 +186,276 @@ <h3>Selected</h3>
     </div>
 </div>
 
-<hr />
+<h2>Example 3: Listbox with group headers</h2>
+<p>ARROW keys move the active descendant <em>and</em> automatically updated the selected state. This behaviour is similar to a radio button group.</p>
```

Comment:
This note doesn't apply for all the examples in this section. I think it can be removed, or perhaps replaced with some text about what's distinct about group headers.